test/python/scheduler/test_basic_scheduler.py

```diff
@@ -776,13 +758,8 @@ def test_inst_map_schedules_unaltered(self):
         insts = sched1.instructions
         self.assertEqual(insts[0][0], 0)  # ShiftPhase at DriveChannel(0) no dt required.
         self.assertEqual(insts[1][0], 0)  # ShiftPhase at ControlChannel(1) no dt required.
-        self.assertEqual(insts[2][0], 0)  # Pulse Ym_d0 of duration 160dt.
-        self.assertEqual(insts[3][0], 0)  # Pulse X90p_d1 of duration 160dt.
-        self.assertEqual(insts[4][0], 160)  # Pusle CR90p_d1_u0 of duration 720dt.
-        self.assertEqual(insts[5][0], 160)  # Pulse CR90p_u0 of duration 720dt.
-        self.assertEqual(insts[6][0], 880)  # Pulse Xp_d0 of duratoin 160dt.
-        self.assertEqual(insts[7][0], 1040)  # Pulse CR90m_d1_u0 of duration 720dt.
-        self.assertEqual(insts[8][0], 1040)  # Pulse CR90m_u0 of duration 720dt.
```

Comment:
Is this test missing any assertions after the change?

Reply:
Not really, the new schedule has 4 pulses to check. Backends have a map `instruction -> pulse_sequence`, and for 2-qubit gates `GenericBackendV2` introduces a sequence of 4 pulses applied on different channels. The sequence is: `pulse_2` on drive channel 0,`pulse_3` on control channel 0, `pulse 2` on drive channel 1 and `shift phase` on drive channel 1, in no particular order because they are all on different channels (and this is why they are all applied at the same time, i.e, `t=0`).  

This explanation made me realize that the comments are wrong (they don't correspond to the pulses being generated), but given the size of the PR I think it's better to leave the comment-fixing to a follow-up after the release candidate.

Reply:
The `_get_calibration_sequence()` method in `GenericBackendV2` is the one that returns the pulse sequence that will be used for each instruction. There are 3 different types of sequences, one for `measure`, one for single-qubit gates and one for the rest. These are all dummy calibrations, I chose the pulse shapes and channels kind of arbitrarily, based on examples I've seen, but the point is to have some pulses to check that the order in which they are applied matches the scheduling approach.